_data/team.yml

```
@@ -10,7 +10,7 @@
 - name: Adriana Zenck Zimmermann
   position: Website Coordinator
   comment: Programme support
-  image: https://media.licdn.com/dms/image/C4E03AQFb3yz9xF917A/profile-displayphoto-shrink_800_800/0/1650927176851?e=1678320000&v=beta&t=soCLefighqi-aVaO40bQxgs4z5tDj9X5ZCD09jaP3as
+  image: assets/images/team/3.jpeg
```

You don't need to upload the images again you can use the previous ones
```suggestion
  image: assets/images/mentors/3.jpeg
```